server/pkg/asset/asset.go

```diff
@@ -13,7 +15,7 @@ type Asset struct {
 	size        uint64
 	previewType *PreviewType
 	file        *File
```

Comment:
```suggestion
```

Asset shoud not have a File. A file has a io.ReadCloser and it cannot be initialized without a connection such as HTTP, and if not released in the correct procedure, it will cause a connection leak. A File struct is only initialized and used in very short time in usecase functions.

Reply:
File in pkg/asset/file is an AssetFile which has the following attributes:
```type AssetFile {
  name: String!
  size: FileSize!
  contentType: String
  path: String!
  children: [AssetFile!]
}```
it is different from pkg/file.

Reply:
Oh, ok